scripts/recan_bundle_tdr.py

```diff
@@ -109,8 +112,9 @@ def content_length(content: JSON) -> int:
     return len(json.dumps(content).encode('UTF-8'))
 
 
-def random_uuid() -> str:
-    return str(uuid.uuid4())
+def deterministic_uuid(bundle_uuid: str, entity_name: str) -> str:
```

Comment:
```suggestion
def deterministic_uuid(bundle_uuid: str, *name: str) -> str:
```

So you can use a separator, and don't have to do all the string concatenation in the call sites. Ensure that the separator does not occur in the keys. That's crucial for the correctness of such hashing schemes.